### ISSUE TITLE
Upgrade eslint v2.x -> v3.x

### ISSUE THREAD

**Issue by jacobq** [sun1x/eslint-to-editorconfig#5]
I've run into errors like the following, and they appear to be due to using an outdated version of `eslint`

```
/path/to/project/node_modules/eslint-to-editorconfig/node_modules/eslint/lib/config/config-file.js:402
            throw e;
            ^

Error: /path/to/project/node_modules/eslint-config-airbnb-base/rules/es6.js:
	Configuration for rule "arrow-parens" is invalid:
	Value "as-needed,[object Object]" has more items than allowed.

Referenced from: airbnb-base
Referenced from: /path/to/project/.eslintrc
```

**Comment by zenflow** [comment]
Current version of eslint is 4.18.2 🙁

**Comment by snanovskyi** [comment]
Hey there!
I released eslint-to-editorconfig v2.0.0 and deprecated old versions.
New version supports `eslint>=2.0.0` and works as intended. Check [README](https://github.com/sun1x/eslint-to-editorconfig/blob/master/README.md) if you want to give it a shot.
P.S. I was total noob and I almost had no idea what I was doing in old versions 😔
